### ISSUE TITLE
vue-cli3 多入口的postcss-pxtorem配置，我只想在app使用这个px转换rem，有什么好的解决方案嘛？

### ISSUE THREAD

**Issue by 80sky** [youzan/vant#2643]
![image](https://user-images.githubusercontent.com/22342011/51812365-26cb6180-22ec-11e9-85aa-3efb23a0e593.png)
![image](https://user-images.githubusercontent.com/22342011/51812383-364aaa80-22ec-11e9-83cb-6d7e732f530c.png)

vue创建的多模块的，只想在app使用px转换rem，求大佬们指点，谢谢！

**Comment by haxxk** [comment]
px2了解一下？

**Comment by haxxk** [comment]
https://www.jianshu.com/p/b29ecfdc54d4

**Comment by wywppkd** [comment]
## 用pxtorem的selectorBlackList选项排除van-开头的样式
 如果pxtorem官方能提供一个排除文件夹|文件的选项就更好了
```
const autoprefixer = require('autoprefixer')
const pxtorem = require('postcss-pxtorem')
module.exports = {
  plugins: [
    autoprefixer(),
    pxtorem({
      rootValue: 75.0, // 设计稿为750px
      propList: ['*'], // 对所有css属性进行转换rem
      minPixelValue: 2, // 最小像素值
      selectorBlackList: ['van-'] // 排除van-开头(即vant库中的css样式)的class名
    })
  ]
}
```

**Comment by wywppkd** [comment]
试一下另一个px2rem插件看看有没有配置解决，貌似可以排除整个node_module目录，我没用过这个插件




    



    
    
    



    
        
            
                
                    
                        
                    
                    
                        
                            1055025755
                            
                            
                                
                                    邮箱：1055025755@qq.com
                                
                        
                    
                
            
        
    


Signature is customized by Netease Mail Master


在2019年02月03日 11:47，neverland 写道： Closed #2643.

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/youzan/vant","title":"youzan/vant","subtitle":"GitHub repository","main_image_url":"https://github.githubassets.com/images/email/message_cards/header.png","avatar_image_url":"https://github.githubassets.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/youzan/vant"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"Closed #2643."}],"action":{"name":"View Issue","url":"https://github.com/youzan/vant/issues/2643#event-2114135667"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/youzan/vant/issues/2643#event-2114135667",
"url": "https://github.com/youzan/vant/issues/2643#event-2114135667",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by xiaxiaoxiaobaby** [comment]
解决了吗，我和你一样的问题
